newm-server/src/main/kotlin/io/newm/server/features/minting/repo/MintingRepositoryImpl.kt

```diff
@@ -923,45 +923,34 @@ class MintingRepositoryImpl(
         mapItemKey = "artists".toPlutusData()
         mapItemValue =
             plutusData {
+                val primaryArtistEmails =
```

Comment:
We are only using the user's stageOrFullName if there aren't any other collaborators that are credited. We need at least one credited artist.